experimental/telemetry/doc.go

```diff
@@ -15,4 +15,30 @@
 // If you want to use your own data plane URL, add also this line and
 // make sure the MM_RUDDER_DATA_PLANE_URL environment variable is set.
 //   LDFLAGS += -X "github.com/mattermost/mattermost-plugin-api/experimental/telemetry.rudderDataPlaneURL=$(MM_RUDDER_DATA_PLANE_URL)"
+//
+// In order to use it you should:
+// 1. Start the telemetry client on plugin activate:
```

Comment:
Should the fields that need to be added to `Plugin` also get documented?

Reply:
0/5. I think that is obvious you need to add them to the plugin there. And the actual need is not to have them declared on plugin, but have them declared. Where they get stored is up to the implementation.

Reply:
Sure it's up to the plugin, but this documentation provides an opinionated suggestion. Having the explicit field names here could help with keeping the code bases in sync and lowering the barrier for someone to start using telemetry. 1/5